JSONParser.cpp

```diff
@@ -0,0 +1,79 @@
+#include "JSONParser.h"
+
+void JSONParser::cleanJSONWord(std::string& text) {
+	int firstQuotMPos = text.find('"');
+
+	if (firstQuotMPos >= 0) {
+		int lastQuotMPos = text.find('"', firstQuotMPos + 1);
+		text = text.substr(firstQuotMPos + 1, lastQuotMPos - (firstQuotMPos + 1));

```

Comment:
Teszteltél pl. páratlan számú idézőjelre?

Reply:
@wajzy Ha hasonló hibát talál a json-ban, akkor kerülje meg, és olvassa be az adatokat, vagy dobjon exceptiont?

Reply:
Néha értelmes döntés megpróbálni átlépni a hibán, és feldolgozni, amit lehet. Ezt teszik pl. a fordítóprogramok vagy a webböngészők, de most itt nem szeretném hosszan kifejteni, miért.
Többnyire azonban tisztább a megoldás, ha jelezzük az első hibát értelmesen, hogy rájöjjön a user, mi a baj, és hogyan tudja azt elhárítani, majd leállunk. Ezt pl. kivételek dobásával jól meg lehet valósítani, én is ezt javaslom.